learn_script.py

```diff
@@ -160,7 +175,7 @@ def main():
         for _, fname in invalid_files:
             files.remove(fname)
             ansiprint(
-                "<red>Warning - '{}' not found!</red>\n\n".format(fname), end="",
+                "<red>Warning - '{}' not found!</red>\n\n".format(fname)
```

Comment:
i missed that double `\n`. without end="" there will be three new empty lines.

avoid `\n` and use something like:

```suggestion
                "<red>Warning - '{}' not found!</red>".format(fname)
```